### PR TITLE
Minor rewrite of `std::io::empty` doc example.

### PR DESCRIPTION
**Pull request by frewsxcv**

### CODE REVIEW

File: src/libstd/io/util.rs [hunk 1]
```diff
@@ -78,14 +78,15 @@ pub struct Empty { _priv: () }
 /// A slightly sad example of not reading anything into a buffer:
 ///
 /// ```
-/// use std::io;
-/// use std::io::Read;
+/// use std::io::{self, Read};
 ///
 /// # fn foo() -> io::Result<String> {
 /// let mut buffer = String::new();
 /// try!(io::empty().read_to_string(&mut buffer));
```
Comment (sfackler) [inline src/libstd/io/util.rs]: Should all these `try!`s be converted to unwraps? That seems to be the modern convention?

Comment (frewsxcv) [inline src/libstd/io/util.rs]: Works for me. I'll change them.

Comment (frewsxcv) [inline src/libstd/io/util.rs]: Fixed in the latest force push.

**Comment by GuillaumeGomez** [comment]
Thanks!

@bors: r+ rollup

**Pull request merged**